packages/release-notes-generator/src/index.ts

```diff
@@ -15,15 +15,10 @@ process.on('beforeExit', async () => {
 async function run() {
 	const { mainVersion, isPrerelease, prereleaseId, packageVersions } = await processPackages();
 
-	// Run after `processPackages` to allow package clean-up
-	if (changesets.size === 0) {
-		earlyExit();
-	}
-
 	const { types, untypedPackages, notices } = await getInfo(changesets);
 
 	if (types.length === 0 && untypedPackages.length === 0 && packageVersions.length === 0) {
-		earlyExit();
+		process.stderr.write('WARN: No (processable) changesets found\n');
```

Comment:
Does it no longer have to exit after throwing this error?

Reply:
I realized `changesets` itself would already exit before calling `release-notes-generator` in case no changesets are found, therefore the removal of 
https://github.com/directus/directus/blob/d423fe61dc4c7dc9f277bb22a2963d83b83ef982/packages/release-notes-generator/src/index.ts#L18-L21

Now, there might be cases where there are no release notes entries (this is what this check does) but we still want to release a new version. So I think a hard exit would never have been necessary, but a warning is still appropriate.